webui/react/Makefile

```diff
@@ -22,7 +23,7 @@ get-deps: check-requirements node_modules/done.stamp
 .PHONY: build
 build: check-requirements build/done.stamp
 
-build/done.stamp: $(source_files) node_modules/done.stamp tsconfig.json craco.config.js jest.config.js
+build/done.stamp: $(source_files) node_modules/done.stamp tsconfig.json craco.config.js jest.config.js vite.config.ts
```

Comment:
question: probably part of the plan but just wanting to confirm that we are removing craco from this PR. `craco.config.js`, craco packages, etc

Reply:
We're keeping CRACO for now because it's handling our tests. While we could migrate to Vitest or ava or stay with jest, changing the testing framework would require modifying all of our preexisting tests (even if we ejected from craco and stayed with jest, we'd likely want to ensure that it used the transpilation library we use in our build, which would require changes to our tests due to mocking behavior changes stemming from the difference between es modules and commonjs modules).

Reply:
thanks makes total sense